src/clj/witan/send/model/output.clj

```diff
@@ -162,23 +162,22 @@
 (defn dirichlet-expectations
   "Dirichlet distro parameters are stored in a map typically keyed by an entities index (ay, n, s)."
   [alphas]
-  (-> (reduce (fn [acc [k params]]
-                (concat acc
-                        (let [prefix (if (vector? k)
-                                       (let [v (states/split-entity k)
-                                             ay+1 (inc (first v))]
-                                         (conj v ay+1))
-                                       [k])
-                              normalisation (normalisation-constant-dirichlet params)]
-                          (map (fn [p] (into []
-                                             (concat prefix
-                                                     (conj (mapv name (states/split-need-setting (key p)))
-                                                           (/ (val p) normalisation)
-                                                           normalisation
-                                                           (val p)))))
-                               params))))
-              [] alphas)
-      (sort)))
```

Comment:
How come?

Reply:
it was getting a mix of string and number types in one of the fields (I didn't quite track it down). It doesn't seem to affect any results if you don't care about sorting as it writes it out to csv.